source/loader.c

```diff
@@ -53,9 +53,7 @@ void load_from_file(FILE *f, char *arg) {
     argv.argv = &argv.commandLine;
     argv.endARGV = argv.argv + 1;
 
-    struct stat st;
     int fd = fileno(f);
-    if (fstat(fd, &st)) return;
```

Comment:
Any reason to remove this?  I can't remember, but assume it was probably there for a reason :/